code/modules/necromorph/corruption/growth.dm

```diff
@@ -67,10 +67,10 @@
 
 
 /datum/signal_ability/placement/corruption/branch
-	name = "branch"
+	name = "Branch"
 	id = "branch"
 	desc = ""
-	energy_cost = 135
+	energy_cost = 120 // 135 -> 120, 25th of Aug, 2022
```

Comment:
if we ever decide to get previous cost we will use commit histroy. just remove the comment